pico_w/wifi/dtls/README.md

```diff
@@ -0,0 +1,65 @@
+# Setup
+
+These examples demonstrate how to use dtls via mbedtls on a Pico W device.
+You need to define DTLS_SERVER and run the makecerts.sh script to generate the certificates and keys needed for the server and client.
+```
+export DTLS_SERVER=myserver
+cd dtls/certs
+./makecerts.sh
+```
+The examples should now build.
+
+# Running the dtls examples
+
+The client connects to a server and sends it a few lines of text which it expects to be sent back.
+
+You can build and run the client and server examples on two Pico W devices. To make testing easier to test with just one Pico W device, you can run the server or client on a Linux host.
+The client.sh and server.sh scripts show how to run the client or server with openssl. The host folder contains source code for a version of the client and server using mbedtls.
```

Comment:
This seems to be repeated in the next section?

Reply:
Removed some duplication.